modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/rel/AggregateNode.java

```diff
@@ -95,6 +95,13 @@ public AggregateNode(ExecutionContext<Row> ctx, RelDataType rowType, AggregateTy
         }
 
         grpSet = b.build();
+
+        if (type == AggregateType.REDUCE || type == AggregateType.SINGLE)
```

Comment:
It's better to move this to `Grouping` constructor. And please leave a few words describing why don't we do the same for MAP aggregate

Reply:
Also curly braces is required for `if` since its body is not one liner